### PR TITLE
Recipies list

### PR DESCRIPTION
**Pull request by ZahraArshia**
Hello guys 😁 
## in this step of the project:
- a list of recipes created by the logged-in user can be displayed now!
- each reprice in the recipes list leads to recipe details
- If the user is the owner of the recipe,  user have the permission to delete it.

### CODE REVIEW

**Review by Beardless-sheik — CHANGES_REQUESTED** [review]
Great work on this PR @ZahraArshia . I only have one concern about the UI of the recipe addition view. It would be great to give the User a better description of the boolean input added.

File: app/views/recipes/new.html.erb [hunk 1]
```diff
@@ -1,3 +1,10 @@
-<h1>Recipes#new</h1>
-<h1>add new recipes</h1>
-<p>Find me in app/views/recipes/new.html.erb</p>
+<div class="recipe-form">
+  <%= form_with model: @recipe, url: recipes_path do |form| %>
+      <%= form.text_field :name, placeholder: 'Name'%>
+      <%= form.number_field :preparation_time, placeholder: 'Preparation Time' %>
+      <%= form.number_field :cooking_time, placeholder: 'Cooking Time' %>
+      <%= form.text_area :description, placeholder: 'Description' %>
+      <%= form.select :public, options_for_select([true, false]) %>
```
[the remark this answers is not in the document]
Comment (ZahraArshia) [reply app/views/recipes/new.html.erb:7]: OK sure, I haven't added any styles yet

[the remark this answers is not in the document]
Comment (Beardless-sheik) [reply app/views/recipes/new.html.erb:7]: Ok cool. We can refine it when we add the styles then!

**Review by Beardless-sheik — APPROVED** [review]
Great! @ZahraArshia we can merge and move onto the next milestone.

**Pull request merged**